samples/wifi/shell/overlay-zperf.conf

```diff
@@ -12,7 +12,7 @@ CONFIG_NET_PKT_RX_COUNT=30
 CONFIG_NET_PKT_TX_COUNT=29
 CONFIG_NET_BUF_RX_COUNT=30
 CONFIG_NET_BUF_TX_COUNT=58
-CONFIG_HEAP_MEM_POOL_SIZE=230000
+CONFIG_HEAP_MEM_POOL_SIZE=180000
```

Comment:
why is this change needed? No mention of it in the commit log?

P.S. Commit log should explain in just English rather than quoting code or configs